src/components/ClusterOverview/Alerts/Alerts.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+} from '../../Dashboard/DashboardCard';
+import { ClusterOverviewContext } from '../ClusterOverviewContext';
+import { AlertsBody } from '../../Dashboard/Alert/AlertsBody';
+import { filterAlerts } from './utils';
+
+export const Alerts = ({ alertsResponse }) => {
+  if (!Array.isArray(alertsResponse)) {
+    return false;
+  }
+
+  const alerts = filterAlerts(alertsResponse);
+
+  return alerts.length > 0 ? (
+    <DashboardCard>
+      <DashboardCardHeader>
+        <DashboardCardTitle>Alerts</DashboardCardTitle>
+      </DashboardCardHeader>
+      <DashboardCardBody>
+        <AlertsBody alerts={alerts} />
+      </DashboardCardBody>
+    </DashboardCard>
+  ) : (
+    false
+  );
+};
+
+Alerts.propTypes = {
+  alertsResponse: PropTypes.oneOfType([PropTypes.array, PropTypes.object]),
```

Comment:
can we rather send an empty array from `web-ui` ? This would make things a bit cleaner here

Reply:
Im not sure, even though there is no `Not Available` state yet I think that in the future we will want to notify user that `AlertManager` is not available/accessible and `object` as `alertsResponse` will serve its purpose.